conf/denas/asr/supernet_large.conf

```diff
@@ -29,4 +29,8 @@ SEARCH_SPACE:
     - 240
     - 324
     - 384
-    - 444
\ No newline at end of file
+    - 444
```

Comment:
We should separate the Prunner config with the previous built-in configuration.

Reply:
separated the Pruner config